qutebrowser.json

```diff
@@ -0,0 +1,42 @@
+{
+    "homepage": "https://www.qutebrowser.org/",
+    "license": "https://github.com/qutebrowser/qutebrowser/blob/master/LICENSE",
+    "version": "1.1.1",
+    "persist": [
+        "config"
+    ],
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/qutebrowser/qutebrowser/releases/download/v1.1.1/qutebrowser-1.1.1-windows-standalone-amd64.zip",
+            "hash": "9c9693e7b7c28266b4edcf6de90ae15af3bde4aed4359149a25f6942f3b186a4",
+            "extract_dir": "qutebrowser-1.1.1-x64"
+        },
+        "32bit": {
+            "url": "https://github.com/qutebrowser/qutebrowser/releases/download/v1.1.1/qutebrowser-1.1.1-windows-standalone-win32.zip",
+            "hash": "27a0fae3706c8c4b5381f43b91e57bd3eb0c9f4a6594b97d84c22e01211fa44c",
+            "extract_dir": "qutebrowser-1.1.1-x86"
+        }
+    },
+    "bin": "qutebrowser.exe",
+    "shortcuts": [
+        [
+            "qutebrowser.exe",
+            "qutebrowser"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/qutebrowser/qutebrowser"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/qutebrowser/qutebrowser/releases/download/v$version/qutebrowser-$version-windows-standalone-amd64.zip",
+                "extract_dir": "qutebrowser-1.1.1-x64"
```

Comment:
No `$version` variable

Reply:
So much mistake, so much fun! I'm thinking writing a book about "bad habbits and consequences of copy paste mechanisms". Thx for all the support @r15ch13!

Reply:
For the next app you want to add, just copy this file and edit it. It now contains pretty much every part of a scoop manifest (except `post/pre_install`) 😄